engine/src/main/java/org/hibernate/validator/internal/metadata/provider/AnnotationMetaDataProvider.java

```diff
@@ -705,7 +726,119 @@ public ConstraintType getPartition(ConstraintDescriptorImpl<?> v) {
 	 * @return a set of type arguments constraints, or an empty set if no constrained type arguments are found
 	 */
 	protected Set<MetaConstraint<?>> findTypeAnnotationConstraintsForExecutableParameter(Member member, int i) {
-		// To be extended by subclasses
-		return Collections.emptySet();
+		Parameter parameter = ( (Executable) member ).getParameters()[i];
```

Comment:
That's not new code, it just has been inlined from the separate sub-class we had before.